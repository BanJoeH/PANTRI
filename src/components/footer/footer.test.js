import { shallow, render, mount } from "enzyme";
import toJson from "enzyme-to-json";
import Footer from "./footer";

describe("footer", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<Footer />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
