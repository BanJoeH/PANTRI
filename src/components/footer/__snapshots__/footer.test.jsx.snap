// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders without crashing 1`] = `
<div
  className="footer"
>
  <ul
    className="social-section"
  >
    <li
      className="social github"
    >
      <a
        href="https://github.com/BanJoeH/"
      >
        <img
          alt="gitHub link"
          height="30"
          src="github.png"
          width="30"
        />
      </a>
    </li>
    <li
      className="nav-link"
    >
      <a
        className="social"
        href="https://joescript.io"
      >
        Made by Joe
      </a>
    </li>
    <li
      className="social"
    >
      <a
        href="https://www.linkedin.com/in/joescript/"
      >
        <img
          alt="LinkedIn link"
          height="30"
          src="linkedin.png"
          width="30"
        />
      </a>
    </li>
  </ul>
</div>
`;
